src/components/__tests__/ChatStartDialog.test.js

```diff
@@ -30,32 +30,51 @@ function createChat (context, { partnerId }) {
  * Mockup store helper.
  */
 function mockupStore () {
+  const state = () => ({
+    address: 'U123456',
+    balance: 0,
+    passphrase: ''
+  })
   const snackbar = mockupSnackbar()
   const chat = {
+    getters: {
+      isAdamantChat: () => () => false,
+    },
     actions: {
-      createChat: createChat
+      createChat: createChat,
```

Comment:
use ES6 features
`createChat: createChat` -> `createChat`

Reply:
We will fix such cases later with `prettier` across the entire codebase. That part wasn't touched so I suggest leave it as is.